projects/dtt-app/src/app/app.component.ts

```diff
@@ -0,0 +1,271 @@
+
+        this.layersReverse$ = this.projectService.getLayerStream().pipe(
+            map(layers => layers.slice(0).reverse())
+        );
+
+        this.layerListVisible$ = this.layoutService.getLayerListVisibilityStream();
+        this.layerDetailViewVisible$ = this.layoutService.getLayerDetailViewVisibilityStream();
+    }
+
+    private registerIcons() {
+        this.iconRegistry.addSvgIconInNamespace(
+            'vat',
+            'logo',
+            this.sanitizer.bypassSecurityTrustResourceUrl('assets/vat_logo.svg'),
+        );
+
+        // used for navigation
+        this.iconRegistry.addSvgIcon('cogs', this.sanitizer.bypassSecurityTrustResourceUrl('assets/icons/cogs.svg'));
+
+    }
+
+    ngOnInit() {
+        this.mapService.registerMapComponent(this.mapComponent);
+        this.mapIsGrid$ = this.mapService.isGrid$;
+
+        this.middleContainerHeight$ = this.layoutService.getMapHeightStream(this.windowHeight$).pipe(
+            tap(() => this.mapComponent.resize()),
+        );
+    }
+
+    ngAfterViewInit() {
+        this.layoutService.getSidenavContentComponentStream().subscribe(sidenavConfig => {
+            this.rightSidenavContainer.load(sidenavConfig);
+            if (sidenavConfig) {
+                this.rightSidenav.open();
+            } else {
+                this.rightSidenav.close();
+            }
+        });
+        this.projectService.getNewPlotStream()
+            .subscribe(() => this.layoutService.setSidenavContentComponent({component: PlotListComponent}));
+
+        // notify window parent that this component is ready
+        if (parent !== window) {
+            parent.postMessage({
+                type: 'STATUS',
+                status: 'READY',
+            }, '*');
+        } else {
+
+            // handle query parameters directly if it is not embedded and using an auto login
+            this.handleQueryParameters();
+
+        }
+    }
+
+    private setupNavigation(): Array<NavigationButton> {
+        return [
+            // NavigationComponent.createLoginButton(this.userService, this.layoutService, this.config),
+            {
+                sidenavConfig: AppComponent.setupAddDataConfig(),
+                icon: 'add',
+                tooltip: 'Add Data',
+            },
+            {
+                sidenavConfig: {component: OperatorListComponent, config: {operators: AppComponent.createOperatorListButtons()}},
+                icon: '',
+                svgIcon: 'cogs',
+                tooltip: 'Operators',
+            },
+            {
+                sidenavConfig: {component: PlotListComponent},
+                icon: 'equalizer',
+                tooltip: 'Plots',
+            },
+            {
+                sidenavConfig: {component: TimeConfigComponent},
+                icon: 'access_time',
+                tooltip: 'Time',
+            },
+            {
+                sidenavConfig: {component: HelpComponent},
```

Comment:
I guess we have to define a custom help component since we have no accounts here and should provide a different about section.
(cf. GFBio's help)

Reply:
Reworking help is a new task

Reply:
I created a task for it